dashboard/test/controllers/sessions_controller_test.rb

```diff
@@ -333,6 +333,7 @@ class LtiAccountLinkingSignInPageTest < ActionDispatch::IntegrationTest
 
     before do
       Policies::ChildAccount::ComplianceState.stubs(:locked_out?).with(user).returns(user_is_locked_out)
+      user.update(child_account_compliance_lock_out_date: DateTime.now)
```

Comment:
You can also do (above):

```
let(:user) {create(:parent_managed_student, child_account_compliance_lock_out_date: DateTime.now.iso8601)}
```

Reply:
I do think regardless that the field needs to be a string in order to be parsable. hmm. Like, `DateTime.parse(DateTime.now)` I think errors out.

Reply:
You would need to repeat it below as well for the `let(:user) {create(:student, email: user_email)}` line, so maybe your way is better for the sake of brevity.

Reply:
I could swap it to using `let` if you have a strong preference. Otherwise I'll just leave it as-is to benefit from the our json string bag working as intended